physical/cassandra.go

```diff
@@ -72,6 +75,39 @@ func newCassandraBackend(conf map[string]string, logger log.Logger) (Backend, er
 	connectStart := time.Now()
 	cluster := gocql.NewCluster(hosts...)
 	cluster.Keyspace = keyspace
+
+	cluster.ProtoVersion = 2
```

Comment:
🤔 Fair enough, given their goal is to unify the Cassandra connection implementations.